src/patternfly/components/Alert/alert.scss

```diff
@@ -49,8 +49,36 @@
   --pf-c-alert--m-info__icon--BackgroundColor: var(--pf-global--info-color--100);
   --pf-c-alert--m-info__title--Color: var(--pf-global--info-color--200);
 
+  // Inline modification
+  --pf-c-alert--m-inline__title--Color: var(--pf-global--Color--100);
+  --pf-c-alert--m-inline--BoxShadow: none;
+  --pf-c-alert--m-inline--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-alert--m-inline--BorderStyle: solid;
+  --pf-c-alert--m-inline--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-alert--m-inline--BorderRightWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-alert--m-inline--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-alert--m-inline--BorderLeftWidth: 0;
+  --pf-c-alert--m-inline--before--Width: var(--pf-global--BorderWidth--lg);
+  --pf-c-alert--m-inline--before--Top: calc(-1 * var(--pf-c-alert--m-inline--BorderTopWidth));
+  --pf-c-alert--m-inline--before--Height: calc(100% + var(--pf-c-alert--m-inline--BorderTopWidth) + var(--pf-c-alert--m-inline--BorderBottomWidth));
```

Comment:
this works, but you could also remove `height` and use `bottom` instead as `--pf-c-alert--m-inline--before--Bottom: calc(-1 * var(--pf-c-alert--m-inline--BorderBottomWidth));` (the same way you define `top`)